dev-tools/audius-compose

```diff
@@ -818,31 +813,23 @@ def ps(protocol_dir):
     )
 
     for service in services:
-        if service["Service"] == "port-forwarder":
-            continue
+        name = service["Names"]
+        status = service["Status"] or service["State"]
 
-        name = service["Service"]
-        status = service["Health"] or service["State"]
+        ports = service["Ports"]
 
-        ports = {}
-        if service["Publishers"]:
-            for publisher in service["Publishers"]:
-                if publisher["PublishedPort"]:
-                    ports[publisher["PublishedPort"]] = publisher["TargetPort"]
-
-        if service["Service"] in [
+        if service["Names"] in [
             "discovery-provider",
             "discovery-provider-elasticsearch",
         ]:
             replica = int(service["Name"].split("-")[-1])
-        if service["Service"] == "discovery-provider":
-            name = f"{service['Service']}-{replica}"
+        if service["Names"] == "discovery-provider":
+            name = f"{service['Names']}-{replica}"
             ports[5000 + replica - 1] = 5000
-        if service["Service"] == "discovery-provider-elasticsearch":
-            name = f"{service['Service']}-{replica}"
+        if service["Names"] == "discovery-provider-elasticsearch":
+            name = f"{service['Names']}-{replica}"
 
-        ports = sorted(ports.items())
-        ports = ", ".join(f"{target}->{published}" for target, published in ports)
+        name = name.removeprefix("audius-protocol-")
```

Comment:
cut this off on the names since it's fairly repetitive and it makes the viewing extra wide, since this is just a viewing utility thought we could remove it cosmetically

Reply:
I tend to copy paste the name into `docker logs -f <name>` so this is a bit annoying, but i totally get it. will leave u with this feedback & let you make the call! 

maybe we should have `audius-compose logs -f <name>` 😉